app/main.py

```diff
@@ -1,8 +1,50 @@
+from typing import Any
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand: str
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center: int | float,
+            clean_power: int,
+            average_rating: float,
+            count_of_ratings: int
+    ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = round(average_rating, 1)
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, car_list: list) -> int:
+        inc = 0
+        for car in car_list:
+            inc += self.calculate_washing_price(car)
+            self.wash_single_car(car)
+        return inc
+
+    def calculate_washing_price(self, mark: Any) -> int | float:
```

Comment:
You should not use Any if you can specify type more clear
Change throughout the code